codecov.yml

```diff
@@ -10,3 +10,20 @@ coverage:
         # basic
         target: auto
         threshold: 3%
+
+comment:
+  layout: "header, diff, flags"
+  behavior: default
+  require_changes: false
+
+flag_management:
+  default_rules: # the rules that will be followed for any flag added, generally
+    carryforward: true
+    statuses:
+      - type: project
+        target: 85%
+      - type: patch
+        target: 85%
```

Comment:
I noticed ci failed too much, is it too high?
![image](https://github.com/tikv/pd/assets/53859786/639a00d5-8796-4850-8aad-f59f935952c7)